workflows/joint/joint.wdl

```diff
@@ -62,9 +62,15 @@ workflow joint {
     sv_depth_bw: {
       name: "Depth bedgraph"
     }
+    sv_gc_bias_corrected_depth_bw: {
+      name: "GC bias corrected depth bedgraph"
```

Comment:
s/bedgraph/bigwig/

Reply:
Same for the above entry for non-corrected depth bigwig file

Reply:
Fixed in joint.wdl and sawfish.wdl